lib/components/map/default-map.tsx

```diff
@@ -162,47 +175,57 @@ class DefaultMap extends Component {
           <RouteViewerOverlay />
           <TransitVehicleOverlay />
           <StopViewerOverlay />
-          <TransitiveOverlay getTransitiveRouteLabel={getTransitiveRouteLabel} />
+          <TransitiveOverlay
+            getTransitiveRouteLabel={getTransitiveRouteLabel}
+          />
           <TripViewerOverlay />
           <ElevationPointMarker />
 
           {/* The configurable overlays */}
-          {mapConfig.overlays && mapConfig.overlays.map((overlayConfig, k) => {
+          {mapConfig.overlays?.map((overlayConfig, k) => {
```

Comment:
should this "?" go after mapConfig (`mapConfig?.overlays`) ?

Reply:
Fixed component-wide in https://github.com/opentripplanner/otp-react-redux/pull/572/commits/6c56e6bebcc2fa49ded914626746dd6b58205af4.